cfme/tests/intelligence/test_chargeback.py

```diff
@@ -36,7 +36,7 @@ def test_add_new_compute_chargeback():
 def test_compute_chargeback_duplicate_disallowed():
     ccb = new_compute_rate()
     ccb.create()
-    with error.expected('Name already exists'):  # currently cfme just blows up
+    with error.expected('Description has already been taken'):  # currently cfme just blows up
```

Comment:
5.3 still blows up....5.4 did blow up.....it's now been fixed...but the fix didn't align with the code